packaging/linux/run_keybase

```diff
@@ -45,7 +45,7 @@ echo Launching keybase service...
 keybase service &>> "$logdir/keybase.service.log" &
 echo Mounting /keybase...
 kbfsfuse -debug -mdserver mdserver.kbfs.keybase.io:443 \
-  -bserver bserver.kbfs.keybase.io:443 /keybase &>> "$logdir/keybase.kbfs.log" &
+  -bserver bserver.kbfs.keybase.io:443 -log-to-file /keybase &>> "/dev/null" &
```

Comment:
I use `/dev/null` because I didn't want to risk a bug on our end (say a stray `fmt.Printf` we left in by accident) logging something by accident to someone's terminal long after they called `run_keybase`.  What if we have a `keybase.kbfs.out` that's not rotated and should pretty much always be 0 bytes unless there's a bug?

And no, mdserver and bserver are not explicitly needed, but we can do that in a separate PR.  Don't want to change too much right now.